### DISCUSSION TITLE
.bib File permissions

### DISCUSSION THREAD

**Discussion by igravious** [retorquere/zotero-better-bibtex#1864] [Q&A]
Who, or what, sets them?

Mine are coming out `-rw-------` and I need then to be `-rw-r-----`

In fairness it mightn't even be zotero-better-bibtex that's setting them … but if it is I'd love to know where I could tweak 'em.

**Comment by retorquere** [comment]
BBT doesn't explicitly set the permissions. What permissions do exports with `RDF` get? And what is your default umask set to?

**Reply by igravious** [reply]
Thanks for responding. Excuse my ignorance but what are exports with `RDF`? My default umask is `0002`, that's pretty standard, no?

**Reply by retorquere** [reply]
I don't know -- on my Macbook and my Linux system I've never changed them and they're 0022.

With "an export with RDF" I meant to export the same items you currently export to .bib, but to choose "RDF" instead of "Better BibTeX" in the export dialog.
